specification/ml/_types/TrainedModel.ts

```diff
@@ -121,6 +123,11 @@ export class TrainedModelSizeStats {
 export class TrainedModelDeploymentNodesStats {
   /** The average time for each inference call to complete on this node. */
   average_inference_time_ms: double
+  /**
+   * The average time for each inference call to complete on this node
+   * in the last minute. @Since 8.2.0
```

Comment:
Annotation should be `@since 8.2.0`. We typically put these on their own line too.